cfgov/legacy/templates/complaint/complaint-landing.html

```diff
@@ -204,9 +242,9 @@ <h2>Use our data</h2>
               <p>Looking to export our data and have more control over it? Use our Open Data API.</p>
               <ul class="m-list m-list__links">
                 <li class="m-list_item">
-                  <a class="a-link api-documentation" href="https://dev.socrata.com/foundry/#/data.consumerfinance.gov/jhzv-w97w">
+                  <a class="a-link api-documentation" href="https://cfpb.github.io/api/ccdb/">
                     <span class="a-link_text">
```

Comment:
This span is superfluous. It also gets added by the Python icon-adding middleware.

Reply:
On this legacy page, it appears to add the underline. Without it, the underline is not present.